src/main/resources/META-INF/spring/database.properties

```diff
@@ -44,4 +44,7 @@ flyway.database.username=${ACAS_USERNAME}
 client.containerInventory.search.maxResult=1000
 
 cmpdreg.database.configuration.externalLocation=false
-cmpdreg.database.configuration.configJsonLocation=/CmpdReg/src/client/custom/configuration.json
+cmpdreg.database.configuration.configJsonLocation=/home/runner/build/conf/compiled/conf.js
```

Comment:
Is this even used anywhere? Couldn't find where this gets referenced

Reply:
No, its not used anymore.  I moved it temporarily when starting this work but forgot to remove it later. I'll remove this line.